.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,6 +17,9 @@ If you can, provide a minimal chunk of code that shows the problem (either inlin
 Please attach the clangd stderr log if you can. (Usually available from the editor)
 If possible, run with `--log=verbose` - note that the logs will include the contents of open files!
 If this is a crash, try to put `llvm-symbolizer` on your PATH per the troubleshooting instructions.
+(If you're using Windows, place the associated PDB file (debug symbols) in the same directory as
```

Comment:
It would be great to add this information to https://clangd.llvm.org/troubleshooting#getting-stacktraces-from-crashes as well (the repo is at https://github.com/llvm/clangd-www/blob/main/troubleshooting.md).

Reply:
Thanks, I will put up a PR then.